types/plotly.js/index.d.ts

```diff
@@ -186,6 +186,80 @@ export interface DownloadImgopts {
 	filename: string;
 }
 
+export interface LayoutPolar {
+	domain: {
+		x: number[];
+		y: number[];
+		row: number;
+		column: number;
+	};
+	sector: number[];
+	hole: number;
+	bgcolor: Color;
+	radialaxis: {
```

Comment:
all of these nested properties should be partials...

```ts
radialaxis: Partial<{
  visible: boolean;
  range: any[];
}>;

Reply:
I made the radialaxis into a partial. I'm not super familiar with using Partials - should I make all the nested features within it also a partial (e.g. `title`)?